tests/lib/rules/id-length.js

```diff
@@ -126,27 +138,155 @@ ruleTester.run("id-length", rule, {
             ]
         },
         {
-            code: "var { x} = {};",
+            code: "function foo({x}) { }",
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError
+            ]
+        },
+        {
+            code: "function foo({x: a}) { }",
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                {
+                    messageId: "tooShort",
+                    data: { name: "a", min: 2 },
+                    type: "Identifier"
+                }
+            ]
+        },
+        {
+            code: "function foo({x: a, longName}) { }",
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError
+            ]
+        },
+        {
+            code: "function foo({ longName: a }) {}",
+            options: [{ min: 3, max: 5 }],
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError
+            ]
+        },
+        {
+            code: "function foo({ prop: longName }) {};",
+            options: [{ min: 3, max: 5 }],
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooLongError
+            ]
+        },
+        {
+            code: "function foo({ a: b }) {};",
+            options: [{ exceptions: ["a"] }],
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                tooShortError
+            ]
+        },
+        {
+            code: "function foo({ a: { b: { c: d, e } } }) { }",
```

Comment:
Not a blocker for this PR, but it seems like it would be useful to include the location information in the errors for tests like this, to ensure that it's warning about the correct identifier.

Reply:
@kaicataldo 
you are right! 👍I added. thanks

Reply:
Thank you!